srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9429594.3
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429594.15
srun: Terminating job step 9429594.16
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9429594.7
2160
2160
2160
srun: Terminating job step 9429594.6
srun: Terminating job step 9429594.5
2160
2160
srun: Terminating job step 9429594.4
2160
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9429594.14
srun: Terminating job step 9429594.17
srun: Terminating job step 9429594.18
srun: Terminating job step 9429594.19
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 871, in <module>
    delta = all_instances[instance_index][0]
    main()
IndexError: list index out of range
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429594.12
srun: Terminating job step 9429594.11
srun: Terminating job step 9429594.8
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429594.10
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9429594.9
srun: Terminating job step 9429594.13
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: Terminating job step 9429594.0
srun: Terminating job step 9429594.2
srun: Terminating job step 9429594.1
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n35: task 0: Exited with exit code 1
srun: error: sh03-01n35: task 0: Exited with exit code 1
srun: Terminating job step 9429594.24
srun: Terminating job step 9429594.23
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: Terminating job step 9429594.22
srun: Terminating job step 9429594.21
srun: Terminating job step 9429594.20
2160
2160
2160
2160
2160
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: Terminating job step 9429594.29
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: Terminating job step 9429594.28
srun: Terminating job step 9429594.25
srun: Terminating job step 9429594.26
srun: Terminating job step 9429594.27
